src/test/scala/nl/knaw/dans/easy/solr/ConfSpec.scala

```diff
@@ -0,0 +1,107 @@
+package nl.knaw.dans.easy.solr
+
+import java.io.{ByteArrayOutputStream, File}
+import java.lang.System.{clearProperty, setProperty}
+import java.net.MalformedURLException
+
+import org.apache.commons.configuration.PropertiesConfiguration
+import org.apache.commons.io.FileUtils.{readFileToString, write}
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.JavaConverters._
+import scala.runtime.BoxedUnit
+
+class ConfSpec extends FlatSpec with Matchers {
+
+  // by default tests don't use any cfg/application.properties
+  clearProperty("app.home")
+
+  "options in help info" should "be part of README.md" in {
+    // clearProperty("app.homedir") does not help a maven build so this test should come first
+    val readme = readFileToString(new File("README.md"))
+    val options = printHelp().split("Options:")(1)
+    if (!trim(readme).contains(trim(options)))
```

Comment:
forgot about that one after all the should's (which would not give the desired error message in this case), so yes I can